.github/workflows/cibuildwheel.yml

```diff
@@ -64,6 +64,17 @@ jobs:
         run: |
           cp setup.cfg.template setup.cfg
 
+      - name: Build wheels for CPython 3.10
+        run: |
+          python -m cibuildwheel --output-dir dist
+        env:
+          CIBW_BUILD: "cp310-*"
+          CIBW_MANYLINUX_X86_64_IMAGE: manylinux2010
+          CIBW_MANYLINUX_I686_IMAGE: manylinux2010
```

Comment:
I guess there's no point to use manylinux1 for Python 3.10 now?

Reply:
Yes, Python 3.10 does not support manylinux1 and it was failing with it.

Reply:
There's actually no point to use manylinux1 for 3.9 either - the last reason for manylinux1 was the weird hack of attaching pip 9 to `python3.8-dev` on Ubuntu 18.04. NumPy does not ship manylinux1 wheels for 3.9 anymore.

Reply:
The 3.10 wheels for numpy are manylinux2014. I suppose here we should do the same?